### PR TITLE
Copy Edits (2nd pass)

### PR DESCRIPTION
**Pull request by clyde-johnston**

### CODE REVIEW

File: docs/datasheet/Komar.tex [hunk 1]
```diff
@@ -15,50 +15,55 @@
...
-use in propulsion systems of light unmanned aerial (UAV), underwater, and surface vehicles. State-of-the-art vector
-control algorithms make Komar one of the most energy-efficient ESC on the market. Komar is compatible with virtually
-any PMSM or BLDC motor as it is able to measure the parameters of the driven motor and automatically optimize
-itself for best performance. Komar is based on \mbox{mitochondrik} control module.
-All hardware is open source and available on Github\footnote{\url{https://github.com/Zubax/Komar}}.
+Zubax Komar is a high-quality FOC ESC based on \text{T\'elega} motor control technology. Komar is designed to
+support the propulsion systems of light unmanned aerial vehicles (UAVs), unmanned underwater vehicles (UUVs)
+and unmanned surface vehicles (USVs). State-of-the-art vector control algorithms make Komar one of the most
+energy-efficient ESCs on the market. Komar is compatible with almost all PMSM and BLDC motors and can
+optimize motor performance by automatically calibrating the motor parameters. Komar is based
```
Comment (dimracer) [inline docs/datasheet/Komar.tex]: optimize motor control performance

**Review by dimracer — CHANGES_REQUESTED** [review]

File: docs/datasheet/communication_interfaces.tex [hunk 2]
```diff
@@ -29,8 +29,10 @@ \chapter{Communication interfaces}
 \section{CAN bus}
 The device is equipped with a doubly redundant ISO 11898-2 CAN 2.0A/B interface.
 Each CAN interface has two standard UAVCAN Micro connectors\footnote{Refer to \url{http://uavcan.org}
-for more information on UAVCAN.} that are joined in parallel. Note that the device does not 
-terminate the CAN bus internally.
+for more information on UAVCAN.} that are joined in parallel.
+Please note that the CAN bus must be terminated externally since it is not terminated internally by
```
Comment (dimracer) [inline docs/datasheet/communication_interfaces.tex]: since it can not be terminated internally by the device.

**Review by dimracer — CHANGES_REQUESTED** [review]

File: docs/datasheet/communication_interfaces.tex (hunk 2 above)
Comment (clyde-johnston) [reply docs/datasheet/communication_interfaces.tex]: Done

**Review by dimracer — APPROVED** [review]

**Comment by pavel-kirienko** [comment]
I have submitted a review to Myxa; I suggest transferring relevant changes here as well. Please request another review when done.

**Comment by j3qq4hch** [comment]
[komar drawings.zip](https://github.com/Zubax/Komar/files/4739904/komar.drawings.zip)
Here is a set of updated graphics for V0.3 of the hardware.

**Comment by pavel-kirienko** [comment]
Closing due to the pending rewrite. Expect a PR later.

**Pull request closed without merging**